extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -173,7 +173,7 @@ export const ManageAssetRows = ({
       navigateTo(ROUTES.trustlineError);
     } else if (submitStatus === ActionStatus.SUCCESS) {
       dispatch(resetSubmission());
-      navigateTo(ROUTES.trustlineError);
+      navigateTo(ROUTES.account);
```

Comment:
idk why this was trustlineError before 🤔  it should go back to home on success

Reply:
ooh yeah - good catch!